app/models/auth_app_configuration.rb

```diff
@@ -5,7 +5,6 @@ class AuthAppConfiguration < ApplicationRecord
 
   belongs_to :user
 
-  validates :user_id, presence: true
```

Comment:
QQ: why does this need to be removed?

Reply:
Some changes in AR that cause issues with the factories.

Issue basically looks like this:

```ruby
user = User.build(some: :user_data)
user.phone_configuartions = [PhoneConfiguration.build(some: :phone_data)
user.save!
# Raises and invalid record error because `user_id` chan't be blank on PhoneConfiguration
```

Reply:
Ah, does that mean we should have the validates on the relation instead of on the attribute?

Reply:
Yes, and it is there by default. You have to throw an `optional: true` in there to skip it. In some places we explicitly set `user_id` though so I had to leave some in to get it to work.